deployment/v3/external/msg-gateway/install.sh

```diff
@@ -15,29 +15,38 @@ function msg_gateway() {
   echo Istio label
   kubectl label ns $NS istio-injection=enabled --overwrite
 
-  HOST=mock-smtp.mock-smtp
-  PORT=8025
-  USER=
-  SECRET="''"
+  SMTP_HOST=mock-smtp.mock-smtp
+  SMS_HOST=mock-smtp.mock-smtp
+  SMTP_PORT=8025
+  SMS_PORT=8080
+  SMTP_USER=
+  SMS_USER=
+  SMTP_SECRET="''"
+  SMS_SECRET="''"
 
   read -p "Would you like to use mock-smtp (Y/N) [ Default: Y ] : " yn
+  read -p "Would you like to use mock-sms (Y/N) [ Default: Y ] : " yn
```

Comment:
How this will help?
It is not fulfilling the mock-smtp no and mock sms yes scenerio.
Please get the logic corrected.